src/js/popup/popup.js

```diff
@@ -577,6 +507,86 @@
               popup.panel.masks.search.init();
             }
 
+            // Remove loading state
+            document.body.classList.remove("is-loading");
```

Comment:
Context: Moved this to the end of the `buildMasksList()` function. Previously, we were removing the loading state too early.